lib/src/model.dart

```diff
@@ -178,13 +178,18 @@ abstract class ModelElement {
       return docs;
     }
 
-    String _getMatchingLink(String codeRef) {
+    String _getMatchingLink(String codeRef, [bool isConstructor = false]) {
```

Comment:
Did think about making it named, but was not sure of the convention. Will change it, more readable the better!